rfc/rfc004-verifiable-transactional-graph.md

```diff
@@ -136,15 +136,24 @@ until queue empty; take transaction from queue
     process transaction
 ```
 
-### 3.6. Signature and transaction content verification
+### 3.6 Signature verification
 
-Before interpreting a transaction's content the JWS' signature MUST be validated. When **kid** is used to specify the signing key and the system knows additional usage restrictions \(e.g. the key is valid from X to Y, to be checked against **sigt**\), the system MUST assert the usage is compliant.
+Before interpreting a transaction's content the JWS' signature MUST be validated. Almost all transactions will use the **kid** to identify the key used to sign the transaction.
+Resolving the **kid** based on the signature timestamp (**sigt**) can cause problems and even adds attack vectors.
```

Comment:
I think the **kid** requirement can be part of section 3.1 where (additional) requirements on JWS fields are stated?

Reply:
3.1 is more on required fields and this kid requirement is quite lengthy. The entire requirement therefore got its own paragraph